File: 2.0/csharp/contacts/getUniversalDoNotContacts.cs
using CallfireApiClient;
using CallfireApiClient.Api.Contacts.Model;
using CallfireApiClient.Api.Contacts.Model.Request;
using System.Collections.Generic;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        FindUniversalDncsRequest request = new FindUniversalDncsRequest()
        {
            ToNumber = "12135551126",
            FromNumber = "12130021127",
            Fields = "toNumber,inboundCall,outboundCall"
        };
        IList<UniversalDnc> universalDncs = client.DncApi.FindUniversalDncs(request);
    }
}

File: 2.0/java/contacts/getUniversalDoNotContacts.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.contacts.model.UniversalDnc;
import com.callfire.api.client.api.contacts.model.request.FindUniversalDncsRequest;

import java.util.List;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        FindUniversalDncsRequest request = FindUniversalDncsRequest.create()
            .toNumber("12135551126")
            .fromNumber("12130021127")
            .fields("toNumber,inboundCall,outboundCall")
            .build();
        List<UniversalDnc> universalDncs = client.dncApi().findUniversalDncs(request);
    }
}

